test/data-postgres.test.js

```diff
@@ -0,0 +1,82 @@
+const assert = require("assert");
+const chai = require("chai");
+const should = chai.should();
+const { expect } = require("chai");
+
+
+const { generateProducts } = require('../database-postgres/data-generator/generate-products.js');
+const { writeData } = require('../database-postgres/writeToCSV.js');
+const fs = require('fs').promises;
+
+describe('Generate Product Data', () => {
+
+  let record;
+
+  before( async () => {
+    
+    //generate one record
+    record = await generateProducts();
+    
+  });
+
+  it('should return an object', (done) => {
+    expect(record).to.be.an('object');
+    done();
```

Comment:
I don't think `done` is needed here if you don't have any async operations inside this test block

Reply:
Thanks, that was definitely lazy coding on my part. :)